spec/factories/user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    email {"test@email.com"}
+    password {"123456"}
+		first_name { "Test" }
+		last_name { "Tester" }
+		age { 19 }
+  end
+end
```

Comment:
fix code alignment

Reply:
![image](https://user-images.githubusercontent.com/68684063/109815091-3acfb300-7c6a-11eb-9697-a7cf00eb57d5.png)
This is what mine looks like.